### ISSUE TITLE
undefined local variable or method `json'

### ISSUE THREAD

**Issue by Skovy** [guilhermesad/rspotify#61]
`json` does not exist in the `add_tracks!` method

```
NameError: undefined local variable or method `json' for #<RSpotify::Playlist:0x007f970c3df748>
/Users/.rvm/gems/ruby-2.2.2/gems/rspotify-1.15.2/lib/rspotify/base.rb:150:in `method_missing'
/Users/.rvm/gems/ruby-2.2.2/gems/rspotify-1.15.2/lib/rspotify/playlist.rb:146:in `add_tracks!'
```

```ruby
def add_tracks!(tracks, position: nil)
      track_uris = tracks.map(&:uri).join(',')
      url = "#{@href}/tracks?uris=#{track_uris}"
      url << "&position=#{position}" if position

      response = User.oauth_post(@owner.id, url, {})
      @total += tracks.size
      @tracks_cache = nil

      if RSpotify::raw_response
        @snapshot_id = JSON.parse(response)['snapshot_id']
        return response
      end

      @snapshot_id = json['snapshot_id']
      tracks
    end
```

**Comment by luis4ndre** [comment]
Me too...

**Comment by guilhermesad** [comment]
Please update your gem, this was just fixed on latest version (`1.15.3`). Thanks.
